packages/avatar/package.json

```diff
@@ -18,14 +18,14 @@
     "react-lifecycles-compat": "^3.0.2"
   },
   "peerDependencies": {
-    "prop-types": "^15.5.10",
     "react": "^15.4.1 || ^16.3.2"
   },
   "devDependencies": {
     "@hig/babel-preset": "^0.1.0",
     "@hig/eslint-config": "^0.1.0",
     "@hig/scripts": "^0.1.0",
-    "@hig/styles": "^0.1.1"
+    "@hig/styles": "^0.1.1",
+    "prop-types": "^15.6.1"
```

Comment:
Although PropTypes isn't active in production, it still needs to be available at runtime, so it can't be a dev dependency. I agree with it not being a peer dependency though, [like the docs say](https://github.com/facebook/prop-types/blob/master/README.md#how-to-depend-on-this-package).